lib/configuration/transformers/v1Config.js

```diff
@@ -35,21 +36,37 @@ class V1Config {
 
     if (config.issues) {
       output.push(processValidators('issues.*', config.issues, {pass: [], fail: consts.DEFAULT_ISSUES_FAIL, error: consts.DEFAULT_ISSUES_ERROR}))
+      processStale('issues', config.issues, output)
     }
 
     if (config.pull_requests) {
       output.push(processValidators('pull_request.*', config.pull_requests, {pass: consts.DEFAULT_PR_PASS, fail: consts.DEFAULT_PR_FAIL, error: consts.DEFAULT_PR_ERROR}))
       output.push(processValidators('pull_request_review.*', config.pull_requests, {pass: consts.DEFAULT_PR_PASS, fail: consts.DEFAULT_PR_FAIL, error: consts.DEFAULT_PR_ERROR}))
+      processStale('pull_request', config.pull_requests, output)
     }
     if (!config.pull_requests && !config.issues) {
       output.push(processValidators('pull_request.*', config, {pass: consts.DEFAULT_PR_PASS, fail: consts.DEFAULT_PR_FAIL, error: consts.DEFAULT_PR_ERROR}))
       output.push(processValidators('pull_request_review.*', config, {pass: consts.DEFAULT_PR_PASS, fail: consts.DEFAULT_PR_FAIL, error: consts.DEFAULT_PR_ERROR}))
     }
-
     return {mergeable: output}
   }
 }
 
+const processStale = (type, config, output) => {
```

Comment:
just double checking , did we have `stale` in the first version config where we don't need `issues` or `pull_request`?

Reply:
No. But the format is different. In v1, everything either fall under `issues` or `pull_requests` at the parent node. So, there really is no way to specify `stale` without needing either `issues` or `pull_requests`.